ansible/playbooks/roles/backup/tasks/kubernetes.yml

```diff
@@ -49,11 +49,13 @@
 
     - name: Save etcd snapshot
       shell: |
-        docker run \
-        -v "{{ backup_temp_dir.path }}/:/backup/" \
-        --network host \
+        ctr --namespace k8s.io \
+        run \
+        --mount type=bind,src={{ backup_temp_dir.path }}/,dst=/backup/,options=rbind:rw \
+        --net-host \
         --env ETCDCTL_API=3 \
         --rm "{{ etcd_image_name.stdout | trim }}" \
+        etcd \
```

Comment:
This is not an image name, it's a newly created container id, that could be anything because it is temporary. See syntax:
```
USAGE:
   ctr run [command options] [flags] Image|RootFS ID [COMMAND] [ARG...]
   ```